docker-storage-setup.sh

```diff
@@ -422,9 +425,10 @@ reset_extra_volume () {
 }
 
 reset_lvm_thin_pool () {
+  local thinpool_name=${CONTAINER_THINPOOL:-docker-pool}
   if lvm_pool_exists; then
```

Comment:
oh, test 005 only checks that file /etc/sysconfig/docker-storage got removed. And does not check if thin pool docker-pool got removed or not. That's why automated test did not catch this.